editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -91,12 +71,29 @@ export default function Component(props: Props) {
   );
 }
 
-const Tag = styled("div")(({ theme }) => ({
-  backgroundColor: "lightgrey",
+const tagBackgroundColor: Record<string, string> = {
+  [SectionStatus.NotStarted]: "#F3F2F1",
+  [SectionStatus.ReadyToStart]: "#E6E7F3", // team light?
+  [SectionStatus.Completed]: "#00703C",
+};
+
+const tagTextColor: Record<string, string> = {
+  [SectionStatus.NotStarted]: "#505A5F",
+  [SectionStatus.ReadyToStart]: "#222E73", // team dark?
+  [SectionStatus.Completed]: "#FFFFFF",
+};
+
+const Tag = styled("div", {
+  // Configure which props should be forwarded on DOM
+  shouldForwardProp: (prop) => prop !== "title",
+})(({ title, theme }) => ({
```

Comment:
was having a hard time wrapping my head around the syntax for forwarding props to styled components, this was quite helpful: https://dev.to/rasaf_ibrahim/styled-components-in-material-ui-mui-with-styled-utility-3l3j